cluster/manager.go

```diff
@@ -1398,6 +1424,13 @@ func (c *ClusterManager) Remove(nodes []api.Node, forceRemove bool) error {
 	inQuorum := !(c.selfNode.Status == api.Status_STATUS_NOT_IN_QUORUM)
 
 	for _, n := range nodes {
+
+		if id, cerr := c.GetNodeIdFromIp(n.Id); cerr == nil {
```

Comment:
i think "n" is a copy of api.Node, prefer to change nodes[i] if that is referred elsewhere.